elasticgraph-graphql/sig/elastic_graph/graphql/http_endpoint.rbs

```diff
@@ -84,7 +84,7 @@ module ElasticGraph
       @normalized_headers: ::Hash[::String, ::String]?
       def normalized_headers: () -> ::Hash[::String, ::String]
 
-      def content_type: () -> ::String?
+      def mime_type: () -> ::String?
```

Comment:
```suggestion
      @mime_type: ::String?
      def mime_type: () -> ::String?
```

(The lack of a type for the newly introduced instance variable is causing the type checker to fail.)